recompose-composer/src/main/kotlin/recompose/composer/visitor/ComposingVisitor.kt

```diff
@@ -171,6 +172,14 @@ internal class ComposingVisitor : Visitor {
         }
     }
 
+    override fun visitFrameLayout(node: FrameLayoutNode) {
+        val composable = "Box"
+
+        writer.writeCall(composable) {
```

Comment:
Let's maybe use the more verbose version here, like in the other methods. And let's not forget about adding the modifiers!

```Kotlin
val modifier = ModifierBuilder(node)

writer.writeCall(
    name = "Box",
    parameters = listOf(
        modifier.toCallParameter()
    )
) {
   // ...
}
```

Reply:
I will make the changes and update my local.